x-pack/metricbeat/module/aws/terraform.tf

```diff
@@ -46,3 +46,22 @@ resource "aws_s3_bucket_object" "test" {
   bucket  = aws_s3_bucket.test.id
   content = "something"
 }
+
+resource "aws_instance" "test" {
```

Comment:
@jsoriano I see there is a `force_destroy = true` for creating AWS S3 bucket. Do we also need to have some cleanup for other services here? Or all services are destroyed after the test is run?

Reply:
`terraform destroy` fails if `force_destroy` is not used for S3 buckets. This is a flag to avoid removing buckets by mistake on real deployments, and seems specific of these resources. I am not aware of other resources that need it.

Reply:
You can try to run `terraform apply` and `terraform destroy`, and check that resources are properly destroyed.

Reply:
Got it! Thanks!!